etc/rc1

```diff
@@ -50,3 +50,8 @@ for rcdir in $all_dirs; do
     done
 done
 shopt -u nullglob
+
+# N.B. push cleanup commands in the reverse order of execution
+flux admin cleanup-push "flux queue idle --quiet"
+flux admin cleanup-push "flux job cancelall --user=all --quiet -f --states RUN"
+flux admin cleanup-push "flux queue stop --quiet"
```

Comment:
Just a thought - would it be more straightforward to have an append version of `flux admin cleanup`? This is the only current use case of `flux admin cleanup-push", and it requires an "N.B." comment, so it gave me pause.

Reply:
Or, maybe `flux admin cleanup-push` could take a script on stdin and could be called like:
```
flux admin cleanup-push <<EOF
flux queue stop --quiet
flux job cancelall --user=all --quiet -f --states RUN
flux queue idle --quiet
EOF
```

Reply:
I like the stdin idea!